src/js/components/account/user-registration.js

```diff
@@ -145,7 +145,12 @@ export class UserRegistration {
       .done((response, textStatus) => {
         if (textStatus === "success") {
           $form.hide();
-          $success.show();
+          $success
+            .text(
+              "Your details have been submitted successfully. " +
+              "You should receive an email containing a link to click to verify your email address in the next few minutes."
```

Comment:
```suggestion
              "You should receive an email in the next few minutes containing a link to verify your email address."
```

Reply:
Should we mention something here about checking your spam folder? Also, perhaps something like,

"If you have not received an email in 24 hours, please <a href="mailto:support@mail.com">contact support</a>."

Like you did in https://github.com/OpenUpSA/muni-portal-frontend/pull/145

Reply:
Good idea; I'll add the bit about the spam folder. I'll also add the bit about contacting support but rather after 5 or 10 minutes, since if it doesn't show up after 10 mins something is wrong.